lib/src/configuration/better_player_controller_event.dart

```diff
@@ -13,5 +13,11 @@ enum BetterPlayerControllerEvent {
   setupDataSource,
 
   //Video has started.
-  play
+  play,
+
+  //Show skip intro button
+  showSkipIntro,
```

Comment:
nie używasz tych eventów - tj w aplikacji tam gdzie masz implementacje to jeest na podstawie progresu to pobierane, chciaż rozwiązanie z tymi eventami jest spoko, ale też bym to jednak w 100% zamknął w api better playera jako bezkontekstowy feature

Reply:
zostawie to bo jak flage zmieniam w kontrolerze to żeby zaktualizował sie kontroler to musze użyć eventu wtedy będzie miał tez zaktualizowaną tą flage _showSkipIntro